_posts/2025-11-14-building-gift-card-app-with-github-copilot.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: post
+title: "Building a Gift Card Management App with GitHub Copilot: My First Completed Side Project"
+modified: 2025-11-14
+categories: [articles, Howto]
+tags: 
+  - GitHub Copilot
+  - AI-Assisted Development
+  - PWA
+  - Side Projects
+  - Developer Productivity
+comments: true
+---
+
+Like many developers, my life is littered with unfinished side projects. But this time was different. This time, I had GitHub Copilot as my coding companion, and I actually finished. Let me tell you how AI-assisted development helped me build a [gift card management application](https://sunix.github.io/gift-card/) that I actually use every day — and I did it almost entirely without opening an IDE.
```

Comment:
could you include the image of this commic strip from commitstrip.com that illustrate the situation of side projects that are often unfinished:

https://www.commitstrip.com/en/2014/11/25/west-side-project-story/?setLocale=1

<img width="650" height="995" alt="Image" src="https://github.com/user-attachments/assets/081cf041-1ac2-4714-bfbc-fdfb664c3c0c" />